libs/designsystem/src/lib/components/chart/chart.component.ts

```diff
@@ -22,12 +25,17 @@ import { ACTIVITYGAUGE_OPTIONS, ActivityGaugeOptions } from './options/activityg
 })
 export class ChartComponent implements OnInit, OnChanges {
   @Input() data = [];
-  @Input() breaks: Array<Highcharts.XAxisBreaksOptions> = [];
+  @Input() categories: string[] = [];
+  @Input() breaks: Array<XAxisBreaksOptions> = [];
   @Input() height = 300;
   @Input() type: ChartType = ChartType.PIE;
   @Input() description = '';
   @Input() showDataLabels = true;
-  options: Options = {};
+  // tslint:disable-next-line: no-input-rename
+  @Input('options') optionsForOverride: Options;
+  options: Options = {
+    accessibility: {},
```

Comment:
??

Reply:
Null object for referencing it in mergedOptions.accessibility.description
https://www.tutorialspoint.com/design_pattern/null_object_pattern.htm